docs/geedocs/docsrc/answer/fusionTutorial/displayRulesPoint.rst

```diff
@@ -103,48 +101,48 @@ Configuring Display Rules for Point Data
 
          .. note::
 
-            **Note:** You can use some basic HTML tags to format the
+            You can use some basic HTML tags to format the
             Label text, if desired.
 
-      To the left of «\ **COUNTY**\ » enter **<b>**.
-      To the right of «\ **COUNTY**\ » enter **</b><br/><br/>**, and
-      press **Enter**.
-      Adding this HTML code formats the name of the county as bold,
-      followed by two line breaks to separate it from the text that
-      follows (next step).
-
-      Select **NAME** from the **Insert Field** drop-down list.
-      «\ **NAME**\ » appears on the next line of the text box.
-      To the right of «\ **NAME**\ » enter **<br/><br/>**, and press
-      **Enter**.
-      On the next line, enter **Latitude:** , and then select
+      b. To the left of «\ **COUNTY**\ » enter **<b>**.
+      c. To the right of «\ **COUNTY**\ » enter **</b><br/><br/>**.
+      d. Press **Enter**. Adding this HTML code formats the name of the county
+         as bold, followed by two line breaks to separate it from the text that
+         follows (next step).
+
+      #. Select **NAME** from the **Insert Field** drop-down list.
+         «\ **NAME**\ » appears on the next line of the text box.
+
+      a. To the right of «\ **NAME**\ » enter **<br/><br/>**.
```

Comment:
Lettered steps should be indented a level from numbered steps
1. step
    a. step
2. step